pkgs/os-specific/darwin/apple-sdk/frameworks-runtime-deps.nix

```diff
@@ -0,0 +1,33 @@
+# Some frameworks have recursive dependencies - e.g. CoreData depends on
```

Comment:
I am not sure anymore how I solved this but I think I used CoreFoundation instead of Foundation which is also not a good solution.